desktop-src/Debug/symbol-paths.md

```diff
@@ -60,37 +61,30 @@ The symbol handler treats a path element as a cache element if it begins with th
 
 To see how this works, set this search path.
 
-<dl> **.;cache\****myCache***;srv\*\\\\symbols\\symbols**  
-</dl>
+`**.;cache\****myCache***;srv\*\\\\symbols\\symbols**`
 
 The following is a listing of the symbol handler's verbose output while searching for ntdll.pdb, using the above listed search path.
 
-<pre>DBGHELP: .\\ntdll.pdb - file not found
+```
+DBGHELP: .\\ntdll.pdb - file not found
 DBGHELP: .\\dll\\ntdll.pdb - file not found
 DBGHELP: .\\symbols\\dll\\ntdll.pdb - file not found
 SYMSRV: c:\\myCache\\ntdll.pdb\\0F7FCF88442F4B0E9FB51DC4A754D9DE2\\ntdll.pdb not found
 SYMSRV: ntdll.pdb from \\\\symbols\\symbols: 10497024 bytes - copied
 DBGHELP: c:\\myCache\\ntdll.pdb\\0F7FCF88442F4B0E9FB51DC4A754D9DE2\\ntdll.pdb already cached
 DBGHELP: ntdll - private symbols & lines
 c:\\myCache\\ntdll.pdb\\0F7FCF88442F4B0E9FB51DC4A754D9DE2\\ntdll.pdb
-</pre>
-
-<dl> <dt>
+```
 
 The first three lines of output show the symbol handler processing the first path element of ".". This is a standard path element.
-</dt> <dt>
 
 The fourth line shows the symbol handler using the symbol server to look for the file in the second path element of "cache\*c:\\*mySpace*" which is a cache path element.
```

Comment:
Here the stars is italics. And there're stars to separate between paths. It all looks confusing and wrong